t/.gitignore

```diff
@@ -0,0 +1,8 @@
+data/tests/product/
+fork-share.txt
```

Comment:
I do not recall seeing this file. Where does it come from?

Reply:
Just run `make test`. It's in `t/10-terminal.t`. Maybe there should be some cleanup for these files.